install-apptainer-ubuntu.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# exit when any command fails
+set -e
+
+# Install apptainer from release package on amd64
+ARCH=$(arch)
+if [ $ARCH == "x86_64" ]; then
+  DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y wget
+  DEBIAN_PACKAGE_NAME=apptainer_$1_amd64.deb
+  wget https://github.com/apptainer/apptainer/releases/download/v$3/$DEBIAN_PACKAGE_NAME
```

Comment:
`$3` -> `$1` (maybe easier to define variables at the top of the script, so it is clear what each parameter means)

Reply:
Done